engine/helpers.php

```diff
@@ -49,6 +49,11 @@ function em(): EntityManager
     return DB::get();
 }
 
+function restrict_feature(Feature $feature, $uid = null)
```

Comment:
That's great to restrict acces to a page from a user, but we will also need a helper function to directly get a boolean response and that could be easily user in an if clause. Like `check_auth()` but for the features, for example `check_feature(Feature::Message)`. Could be useful !

Reply:
Done! I called is has_feature()